CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Change Log
 
+## [0.24.0](https://github.com/eclipse/lemminx/milestone/37?closed=1) (January 31, 2023)
+
+### Enhancements
+
+ * CodeLens, References, Rename, Diagnostics support for XML references. See [#1435](https://github.com/eclipse/lemminx/pull/1435), [#1427](https://github.com/eclipse/lemminx/issues/1427), [#1432](https://github.com/eclipse/lemminx/pull/1432), [#1452](https://github.com/eclipse/lemminx/pull/1452), [#1367](https://github.com/eclipse/lemminx/issues/1367).
+ * Code action to add missing required elements with RelaxNG. See [#1418](https://github.com/eclipse/lemminx/issues/1418).
+ * Improve the RelaxNG schema validation. See [#1425](https://github.com/eclipse/lemminx/pull/1425).
+ * Support for `textDocument/documentColor`. See [#639](https://github.com/eclipse/lemminx/issues/639).
+
+### Bug Fixes
```

Comment:
- https://github.com/redhat-developer/vscode-xml/issues/851 (submitted to vscode-xml but appiles to lemminx)

Reply:
Should be https://github.com/eclipse/lemminx/pull/1433 then, but yes i'll add it.